src/middlewared/middlewared/job.py

```diff
@@ -551,6 +569,7 @@ def __encode__(self, raw_result=True):
             'state': self.state.name,
             'time_started': self.time_started,
             'time_finished': self.time_finished,
+            'user': username,
```

Comment:
We already have this information in the `credentials` field below

Reply:
I thought maybe it would be nice to have it explicitly listed (for filter usage) for case where `credentials` is token.

Reply:
Per our conversation, I reworked to include in credential dump.